### PR TITLE
Update demoapp deployment

### PR DESCRIPTION
**Pull request by ading1977**
Update `demoapp` deployment with the following changes:

* Change default repository to `turbonomic`
* Change default image tag to `1.0`
* Allow customizing:
  * `grpc.max_connection_age_ms` command line options
  * gRPC debug and trace
  * `locust` service type
* Allow enabling reporting metrics to `Instana` agent

### CODE REVIEW

File: deploy/demoapp/templates/twitter-cass-user.yaml [hunk 1]
```diff
@@ -19,11 +19,29 @@ spec:
...
           value: cassandra
+{{- if .Values.grpc.debug }}
+        - name: GRPC_VERBOSITY
+          value: DEBUG
+{{- if .Values.grpc.trace }}
+        - name: GRPC_TRACE
+          value: {{ .Values.grpc.trace | quote }}
+{{- end }}
+{{- end }}
+{{- if .Values.instana.enabled }}
```
Comment (esara) [inline deploy/demoapp/templates/twitter-cass-user.yaml:39]: not really an issue, but what happens if this is not changed in the app?
I thought instana is supposed to dynamically inject instrumentations without having to change the app/deployment

File: deploy/demoapp/templates/twitter-cass-tweet.yaml [hunk 2]
```diff
@@ -19,11 +19,29 @@ spec:
...
+{{- if .Values.grpc.debug }}
+        - name: GRPC_VERBOSITY
+          value: DEBUG
+{{- if .Values.grpc.trace }}
+        - name: GRPC_TRACE
+          value: {{ .Values.grpc.trace | quote }}
+{{- end }}
+{{- end }}
+{{- if .Values.instana.enabled }}
+        - name: INSTANA_AGENT_HOST
```
Comment (esara) [inline deploy/demoapp/templates/twitter-cass-tweet.yaml:40]: same comment, I thought instana is supposed to dynamically injected

File: deploy/demoapp/templates/twitter-cass-user.yaml (hunk 1 above)
Comment (ading1977) [reply deploy/demoapp/templates/twitter-cass-user.yaml:39]: @esara , this is not about `Instana` instrumentation, this is about configuring network access for the instrumentation, such that applications running inside a kubernetes container can locate the agent and report metrics. 

Please see [Configure Network Access for Monitored Applications](https://www.instana.com/docs/setup_and_manage/host_agent/on/kubernetes/#configure-network-access-for-monitored-applications).

File: deploy/demoapp/templates/twitter-cass-tweet.yaml (hunk 2 above)
Comment (ading1977) [reply deploy/demoapp/templates/twitter-cass-tweet.yaml:40]: Same as above.

**Comment by ading1977** [comment]
Merge this PR as approved by @esara .
```
Endre Sara  10:31 AM
no, I dont
10:31
please close any open issue and feel free to commit as is
10:31
I was trying to understand what a developer needs to do to use instana
10:31
and I was going to bug instana about it
10:32
since they are so proud that the instrumention is injected dynamiclly
10:32
but you are changing the code
10:32
it is not your problem, it is instana's problem
```

**Pull request merged**